src/csync/vio/csync_vio_local_win.cpp

```diff
@@ -189,12 +196,29 @@ std::unique_ptr<csync_file_stat_t> csync_vio_local_readdir(csync_vio_handle_t *d
 
     file_stat->size = (handle->ffd.nFileSizeHigh * ((int64_t)(MAXDWORD)+1)) + handle->ffd.nFileSizeLow;
     file_stat->modtime = FileTimeToUnixTime(&handle->ffd.ftLastWriteTime, &rem);
+
+    std::wstring fullPath;
+    fullPath.reserve(std::wcslen(handle->path) + std::wcslen(handle->ffd.cFileName));
+    fullPath += handle->path;
```

Comment:
Ok, but you depend on the trailing / here -- can you add this as a comment to the `path` member? (I haven't checked whether it is used elsewhere)

Reply:
i'll add a comment